js/deck.js

```diff
@@ -118,7 +118,7 @@ var base = {
     blanks: function(card, hand) {
       return (card.suit === 'army' && !isArmyClearedFromPenalty(this, hand)) ||
         (card.suit === 'land' && card.name !== 'Mountain') ||
-        (card.suit === 'flame' && card.name !== 'Lightning');
+        (card.suit === 'flame' && card.name !== 'Lightning') || card.id === PHOENIX;
```

Comment:
I read this as:

Phoenix is *always* blanked with a flood, unless the flood's penalty is removed.
Other than floods, Phoenix cannot be blanked by any non-flood card, and it cannot contribute to blanking any other card (e.g., if there was a "BLANKED with any FLAME in your hand" card, it would nullify that).

Reply:
My understanding was exactly the same as @brandonscript. That's why I made it so that Great Flood still blanks Phoenix, even though there is a Beastmaster in hand (see the test "Great Flood can still blank a Phoenix since it is a Flood")